ocs_ci/deployment/hub_spoke.py

```
@@ -7376,16 +7401,14 @@ def create_subscription(self):
-            hosted_odf_version = hosted_odf_version.split("-")[-1]
+        df_version = get_df_version(self.name)
+        if any(tag in df_version for tag in ["latest", "stable"]):
+            df_version = df_version.split("-")[-1]
 
-        version_semantic = version.get_semantic_version(hosted_odf_version)
+        version_semantic = version.get_semantic_version(df_version)
 
-        hosted_odf_version = f"{version_semantic.major}.{version_semantic.minor}"
-        subscription_data["spec"]["channel"] = f"stable-{str(hosted_odf_version)}"
+        df_version = f"{version_semantic.major}.{version_semantic.minor}"
+        subscription_data["spec"]["channel"] = f"stable-{str(df_version)}"
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)